mods/ra/maps/exodus/rules.yaml

```diff
@@ -0,0 +1,185 @@
+Player:
+	MissionObjectives:
+	PlayerResources:
+		DefaultCash: 5000
```

Comment:
Actually, the default cash thing was wrong, with the campaign rules, the default is 0. I'm afraid it needs to be there after all.